app/utils/CheckYourAnswersHelper.scala

```diff
@@ -404,7 +404,12 @@ class CheckYourAnswersHelper(userAnswers: UserAnswers)(implicit messages: Messag
         numberOfEntriesType.get) ++
         Seq(customsRegulationType.get) ++
         (userAnswers.get(CustomsRegulationTypePage) match {
-          case Some(CustomsRegulationType.UnionsCustomsCodeRegulation) => Seq(articleType.get)
```

Comment:
this needs to come out

Reply:
This was left when we were discussing technical debt, I changed it back to previous code. Hope if there are any changes related to this I can cover in technical debt ticket.

Reply:
that's not a good way to resolve.. it was ok for testing but not for real code

Reply:
ok, please advise what you are expecting here, what needs to be changed.